frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/ActionItem.tsx

```diff
@@ -0,0 +1,134 @@
+import { IconButton, Tooltip, styled } from '@mui/material';
```

Comment:
```suggestion
import { IconButton, Tooltip } from '@mui/material';
```

Reply:
Sad that biome didn't clean it up...

Reply:
Ok, we have to run yarn lint manually, that was it